app/main.js

```diff
@@ -54,6 +56,34 @@ function createWindow() {
 
   global.mainWindowId = win.id;
 
+  let appIcon = null;
+  app.whenReady().then(() => {
+    const iconExists = fs.existsSync(iconPath);
+    appIcon = new Tray(iconExists ? iconPath : './build/icon.ico');
```

Comment:
Make it so you can double click on the icon to restore the window

Reply:
Done.